duckduckgo/chrome-zeroclickinfo
# move whitelisting to tabManager

**Reviewer:**



## Description:


The options page whitelisting is broken since we got rid of the Sites class. We're also not able to communicate a whitelist status across multiple open tabs with the same domain. 

To fix this I moved the whitelisting method into the tabManager. Instead of setting a sites whitelist status using `tab.site.setWhitelsted` you will use the tabManager which will handle communicating this event to the other open tabs. 

```
tabManager.whitelistDomain({
    list: [name of the whitelist you want to modify]
    domain: domain you want to whitelist
    value: the whitelist value
})
```

## Steps to test this PR:


1. open multiple tabs to the same site
2. whitelist the site in one tab
3. the options page should show it being whitelisted
4. make sure whitelisting worked by reloading
5. check the other open tabs to make sure they're also whitelisted
6. restart the browser, go to the options page, remove the whitelist
7. make sure the domain is now un-whitelisted

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
